config/initializers/simple_form.rb

```diff
@@ -47,7 +51,7 @@
   end
 
   config.wrappers :uswds_text_input, class: 'margin-bottom-4' do |b|
-    b.use :html5
+    b.use :html5_no_aria_required
```

Comment:
There's a bunch more `:html5` in this file, am I correct in thinking we should update them all?

Reply:
I think so. I was focused on text inputs.

Reply:
There's one more:

https://github.com/18F/identity-idp/blob/ae2f04b33a59ee241d24e6777936167adb46e25e/config/initializers/simple_form.rb#L66

Reply:
Thanks! Added that one.